site.yml

```diff
@@ -79,3 +79,26 @@
   - { role: docker, etcd_client_port1: 2379 }
   - { role: swarm, run_as: master }
   - { role: contiv_network, run_as: master }
+
+# Netplugin master
```

Comment:
here is the gist for swarm fix that seems to work: https://github.com/mapuri/ansible/commit/07cc7444256ae85c6cdd0f7c08b2086a10374448

same issue exists with ucp as well. I will change that as well and may be send a combined PR.